io_scene_nif/modules/nif_export/property/texture/__init__.py

```diff
@@ -91,6 +115,33 @@ def get_uv_node(self, b_texture_node):
             raise util_math.NifError(f"Unsupported vector input for {b_texture_node.name} in material '{self.b_mat.name}''.\n"
                                      f"Expected 'UV Map' or 'Texture Coordinate' nodes")
 
+    def get_global_uv_transform_clip(self, b_texture_node):
+        #get the values from the nodes, find the nodes by name, or search back in the node tree
+        x_scale = y_scale = x_offset = y_offset = clamp_x = clamp_y = None
+        #first check if there are any of the preset name - much more time efficient
+        try:
+            combine_node = b_texture_node.id_data.nodes["Combine UV0"]
```

Comment:
We should update documentation with what we want the convention to be.

I like these fallbacks 💯 
We should log the the expected name, to "enlighten" the user what we had expected, but also that we have potentially picked something that they might not have expected.

Reply:
Do you mean log in the sense of putting it in the documentation, or putting a NifLog.debug message in there?

Reply:
I think we should do both.
Add the "expected" name to the documents.
Add a `NifLog.warn()` message in the case we hit the except: , so we let the user know, hey we didn't find what we would have expected, but we did find this other nice thing that looks like what we are looking for, so we going with that instead. This might not be what you wanted to happen.